pyfakefs/fake_filesystem.py

```diff
@@ -247,6 +254,7 @@ def __init__(
         self._add_root_mount_point(total_size)
         self._add_standard_streams()
         self.dev_null: Any = FakeNullFile(self)
+        self._create_temp_dir()
```

Comment:
Perhaps call self.reset to consolidate all these file system updates in one method?

Reply:
You mean from `__init__`? I think I did this before, but as far as I remember the static checker did not like that the member variables have been initialized outside of `__init__`. But you are right, that is too much duplication...